### PR TITLE
Added pull request template

### PR DESCRIPTION
**Pull request by anurag-ks**
Issue Addressed
https://github.com/mayn/packerlicious/issues/46

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## Issue Addressed
+Link to the issue being addressed or any other mentions
+
+## Description
+A few sentences describing the overall goals of the pull request's commits.
+
+## List of Changes Proposed
+
+*
```
Comment (mayn) [inline .github/PULL_REQUEST_TEMPLATE.md]: can you remove this `*`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+
+## Testing
+Specify the steps to test or reproduce the PR
+
+````sh
```
Comment (mayn) [inline .github/PULL_REQUEST_TEMPLATE.md]: remove this example

**Comment by anurag-ks** [comment]
@mayn done.

**Comment by mayn** [comment]
thanks @anurag-ks !

**Pull request merged**